src/backend/api/Fusion.Resources.Database/Entities/DbDepartment.cs

```diff
@@ -0,0 +1,102 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System.Collections.Generic;
+
+namespace Fusion.Resources.Database.Entities
+{
+    public class DbDepartment
```

Comment:
To resolve all sectors, we would just select distinct sector ids?

Reply:
The idea, for now at least, was that sectors would be where DepartmentId == SectorId, but distinct would also work